src/app/(pages)/chatbot/page.tsx

```
@@ -0,0 +1,20 @@
+      sx={{
+        position: 'fixed',
+        left: 0,
+        right: 0,
+        top: '46px',
+        bottom: '28px',
+      }}
+    >
+      <iframe
+        src="https://salvador-airport.strelo.com.br/"
+        style={{ width: '100%', height: '100%' }}
+      />
```

Biome (1.9.4)

[error] Provide a title attribute when using iframe elements.

Screen readers rely on the title set on an iframe to describe the content being displayed.

(lint/a11y/useIframeTitle)